docs/rollup.config.ejs

```diff
@@ -1,23 +1,23 @@
 import commonjs from "@rollup/plugin-commonjs";
-import babel from "rollup-plugin-babel";
+import babel from "@rollup/plugin-babel";
 import builtins from "rollup-plugin-node-builtins";
 import resolve from "@rollup/plugin-node-resolve";
-import {uglify} from 'rollup-plugin-uglify';
+import { terser } from 'rollup-plugin-terser';
```

Comment:
I looked into switching to terser some time ago and found it didn't work. But now I can't remember why. It seems to work now, so maybe the problem was with rollup v1.x. Let's go ahead with this, we can always revert if it causes problems.